core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockSamDAO.scala

```diff
@@ -239,6 +239,46 @@ class MockSamDAO(dataSource: SlickDataSource)(implicit executionContext: Executi
       case _ => Future.successful(Seq.empty)
     }
 
+  override def listResourcesWithActions(resourceTypeName: SamResourceTypeName,
+                                        action: SamResourceAction,
+                                        ctx: RawlsRequestContext
+  ): Future[Seq[SamUserResource]] =
+    resourceTypeName match {
```

Comment:
The content of this mock method is mostly copied from `listUserResources`, with just the action included.  Not sure if we need anything better